Libraries/Image/Image.ios.js

```diff
@@ -118,6 +118,20 @@ var Image = React.createClass({
      * 'stretch': Scale width and height independently, This may change the
      * aspect ratio of the src.
      */
+    getSize: PropTypes.func,
+    /**
```

Comment:
@bestander do you know how the docs get generated? Can we add support for docblocks inside 'statics', so that it's not necessary to mark everything as a prop?